### ISSUE TITLE
[RFC] Featured Extension

### ISSUE THREAD

**Issue by PhillippOhlandt** [bolt/marketplace-site#54]
I thnk it would be nice if we can feature (or let feature via cronjob and random picking) extensions on a daily basis. Right now the Store homepage doesn't change much and many extensions are never really seen by people. I already tried to solve that with my approach of defining extension collections but having a daily featured extension on the homepage would help a bit more I think.

**Comment by GawainLynch** [comment]
Sure … do we just randomly feature one, or do we curate a list?

**Comment by PhillippOhlandt** [comment]
I would randomly pick one via cronjob every day. That way it's fair for everyone. We could also think about a blacklist for old, buggy or unmaintained extensions.

**Comment by GawainLynch** [comment]
> We could also think about a blacklist for old, buggy or unmaintained extensions.

Yeah, that is my concern with "random"

**Comment by PhillippOhlandt** [comment]
Yeah, get the point. A blacklist is much better than a whitelist IMO. A whitelist needs much more maintenance.

**Comment by SahAssar** [comment]
IMO there is no point to have a "featured" feature if it's going to be random...

**Comment by GawainLynch** [comment]
Yeah, it does somewhat dilute the definition … and I am with you there. It also goes to my concerns about promoting "questionable" extensions.

**Comment by PhillippOhlandt** [comment]
Then we need to find a better solution to the problem that the homepage always shows the same extensions and themes.

**Comment by PhillippOhlandt** [comment]
The problem I see when someone picks the featured extension is that the person could be biased and will never pick an extension that he/she doesn't like/use/whatever. This assumption could also include me.

**Comment by GawainLynch** [comment]
Sure… but if someone had the job of curating that list, they could create a short list for the upcoming week and discuss with others in the community… literally in #general

**Comment by PhillippOhlandt** [comment]
hmm, true, true.

**Comment by GawainLynch** [comment]
Thinking further about this idea, if we found an enthusiastic and outgoing type of person, they could really make an "event" out of it each week, even do things like let people know ahead of time that their extension is a candidate and see if they can recruit some voting support 

… sure it would produce a slightly skewed result, but that deviation would likely be the end result of greater community engagement. A good thing :smile:

**Comment by PhillippOhlandt** [comment]
So more something like an "Extension of The Week" thingy where people have to vote the week before? Sounds interesting.

**Comment by GawainLynch** [comment]
Yeah, that sort of thing I guess… a larger part of my point is develop it into something that a community member feels like that have as "their thing" and guide them constructively in founding it, as see where it goes.

…and as @SahAssar said in Slack:

> one way of keeping people invested in bolt is to give them responsibility

**Comment by PhillippOhlandt** [comment]
When we have such an "Extensions of the Week" feature, we should implement it somewhere in the Bolt backend. Either Dashboard or Extend page.
